app/views/questions/identity/edit.html.erb

```diff
@@ -11,27 +11,47 @@
               <%= @main_heading %>
             </h1>
 
-            <div class="question__illustration">
-              <%= image_tag("questions/#{illustration_path}", alt: "") %>
+            <div class="step inherit-link-color">
+              <div class="step-icon-wrapper--large">
+                <%= image_tag("id-card.svg", alt: "") %>
+              </div>
+              <div class="step-text">
+                <p>
+                  We are partnering with <a href="https://www.id.me/individuals/government" target="_blank">ID.me</a> to protect your identity and ensure that your personal information is accurate and secure.
+                </p>
+              </div>
             </div>
 
-            <p>
-              We are partnering with ID.me to protect your identity and ensure that your personal information is accurate and secure.
-            </p>
-
-            <p>
-              You will need to upload your ID and take a selfie, so we can be sure it’s you! It usually takes about 5 minutes to complete.
-            </p>
+            <div class="step">
```

Comment:
do we use BEM naming for CSS styles? I guess this is not relevant to this commit, but a BEM naming convention for this would be `steps__step`, which would also allow the SCSS to be un-nested from within `steps`.

Reply:
ah, i think we haven't been the most diligent about sticking to a naming system - i'd love to go in and clean up some things like this one of these days...